app/controllers/sessions_controller.rb

```diff
@@ -20,7 +20,7 @@ class SessionsController < ApplicationController
 
   def new
   end
-
+  #orRgKyGUs7cz
```

Comment:
What's this?  delete if not needed.  If secret, please add to some other non VCS file using `.gitignore`

Reply:
My bad. it was meant to be deleted. Thanks for spotting this.